src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -33,26 +33,43 @@ class CommitAuthor___commit__count(CommitAuthor):
         commit__count: int
 
 
-filtered_email_domains = {
+FILTERED_EMAIL_DOMAINS = {
     "gmail.com",
+    "yahoo.com",
     "icloud.com",
     "hotmail.com",
     "outlook.com",
     "noreply.github.com",
+    "localhost",
 }
 
-filtered_characters = {"+"}
+FILTERED_CHARACTERS = {"+"}
 
 
 class MissingOrgMemberSerializer(Serializer):
     def serialize(self, obj, attrs, user, **kwargs):
-        return {"email": obj.email, "externalId": obj.external_id, "commitCount": obj.commit__count}
+        formatted_external_id = _format_external_id(obj.external_id)
+
+        return {
+            "email": obj.email,
+            "externalId": formatted_external_id,
+            "commitCount": obj.commit__count,
+        }
 
 
 class MissingMembersPermission(OrganizationPermission):
     scope_map = {"GET": ["org:write"]}
 
 
+def _format_external_id(external_id):
```

Comment:
What if this was typed :O
```suggestion
def _format_external_id(external_id: str | None) -> str | None:
```

Reply:
interesting how mypy doesn't complain :O